platform/viewer/src/routes/WorkList/WorkList.jsx

```diff
@@ -1,7 +1,7 @@
 import React, { useState, useEffect } from 'react';
```

Comment:
Changes here are to split out the study list portion of this file, and then use that study list configuration to actually generate the study list part.
Longer term, fully configuring the child elements of the study list would also be a good idea.  That would allow alternative querying strategies such as querying FHIR ImagingDocument instead, or splitting it up into patient/study lists rather than study/series lists (or perhaps both)